notebooks/279-photo-maker/279-photo-maker.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    class UnetWrapper(torch.nn.Module):
</pre><p>here also wrapping is not necessary, ov.convert_model from nightly is able accept dict as input please take a look on <a href="https://github.com/openvinotoolkit/openvino_notebooks/pull/1630" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/pull/1630</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
you can provide it after conversion

Reply:
Yes, my concern is this method will extend length of codes, and waste the function 'prepare_input_info(input_dict)' defined before.